README.md

```diff
@@ -1,9 +1,502 @@
+
+Layer Values:
+
+- <span id="--upscale-period">--upscale-period</span>:
+  - sets the [upscale-period](#upscale-period) value on the [cli layer](#cli-layer)
+- <span id="--default-uptime">--default-uptime</span>:
+  - sets the [default-uptime](#default-uptime) value on the [cli layer](#cli-layer)
+- <span id="--downscale-period">--downscale-period</span>:
+  - sets the [downscale-period](#downscale-period) value on the [cli layer](#cli-layer)
+- <span id="--default-downtime">--default-downtime</span>:
+  - sets the [default-downtime](#default-downtime) value on the [cli layer](#cli-layer)
+- <span id="--downtime-replicas">--downtime-replicas</span>:
+  - sets the [downscale replicas](#downscale-replicas) value on the [cli layer](#cli-layer)
+- <span id="--explicit-include">--explicit-include</span>:
+  - sets the [exclude value](#exclude) on the [cli layer](#cli-layer) to true, which excludes every workload unless the exclude value on the [workload](#workload-layer) or [namespace](#namespace-layer) layer is set to false. See the [layers concept](#layers) for more details.
+
+Runtime Configuration:
+
+- <span id="--dry-run">--dry-run</span>:
+  - boolean
+  - sets the downscaler into dry run mode, which makes it just print what it would have done, instead of actually doing it
+  - default: false
+- <span id="--debug">--debug</span>:
+  - boolean
+  - makes the downscaler print debug information
+  - default: false
+- <span id="--once">--once</span>:
+  - boolean
+  - makes the downscaler exit after one scan
+  - default: false
+- <span id="--interval">--interval</span>:
+  - [duration](#duration)
+  - sets the time between scans
+  - default: 30s
+- <span id="--namespace">--namespace</span>:
+  - comma seperated list of namespaces (`some-ns,other-ns` or `some-ns, other-ns`)
+  - makes the downscaler only get the specified namespaces
+  - default: all namespaces
+- <span id="--include-resources">--include-resources</span>:
+  - comma seperated list of [scalable resources](#scalable-resources) (`deployments,statefulsets` or `deployments, statefulsets`)
+  - restricts the downscaler to only scale workloads of this resource kind
+  - default: deployments
+- <span id="--exclude-namespaces">--exclude-namespaces</span>:
+  - comma seperated list of namespaces (`some-ns,other-ns` or `some-ns, other-ns`)
+  - makes the downscaler exclude the specified namespaces
+  - default: kube-system, kube-downscaler
+- <span id="--matching-labels">--matching-labels</span>:
+  - comma seperated list of labels with their value (`some-label=val,other-label=value` or `some-label=val, other-label=value`)
+  - makes the downscaler only include workloads which have any label that machtes any of the specified labels and values
+  - default: none
+- <span id="time-annotation">time-annotation</span>:
```

Comment:
I don't fully understand. Is this also a command line argument? Two dashes are missing if so.